### PR TITLE
Setting Up Environment for Book Bot Project

### PR DESCRIPTION
**Pull request by theTytoAlba**
This branch has the following changes:

- Environment is moved to the parent class. Now it is easier to use it for both Book API Homework and Book Bot project.
- Initial setup is done.
- A stub model for ConversationTree is added and its serializer is added. Currently it only contains a title string.

### CODE REVIEW

File: bookbotproject/bookbotproject/settings.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    },
+]
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/1.11/topics/i18n/
+
+LANGUAGE_CODE = 'en-us'
+
+TIME_ZONE = 'UTC'
```
Comment (sevgicansalih) [inline bookbotproject/bookbotproject/settings.py]: You should change Time Zone to "Europe/Istanbul" for consistency.

**Review by sevgicansalih — CHANGES_REQUESTED** [review]
Nicely done!

File: bookbotproject/bookbotproject/settings.py (hunk 1 above)
Comment (theTytoAlba) [reply bookbotproject/bookbotproject/settings.py]: Done!
Thanks for the feedback. :)

**Comment by theTytoAlba** [comment]
Addressed comments. Please take another look!

**Review by sevgicansalih — APPROVED** [review]

**Pull request merged**